### PR TITLE
feat: Lyapunov Exponent Example

### PR DESCRIPTION
**Pull request by FuzzyBunnys**
# Description

This change adds a /Dynamics/ folder  to the .../examples/src/ folder. Included in the /Dynamics/ folders are three files, LyapunovExp.style .substance and .domain implementing an example of a Lyapunov Exponent diagram in Penrose. 

# Implementation strategy and design decisions

The diagram uses 6 randomly generated points to create two curves using the interpolateQuadratic function available in Penrose. The constraints on the diagram are minimal, and it is sometimes necessary to resample a number of times to obtain an example that looks good. 

# Checklist

- The code has minimal comments, happy to edit them however you see fit!

### CODE REVIEW

File: packages/examples/src/registry.json [hunk 1]
```diff
@@ -530,5 +530,8 @@
   "alloy-models/generic": {
     "name": "Alloy: Generic Style",
     "gallery": false
-  }
+  },
+  "Dynamics/Lyapunov": {
+    "name" : "Lyapunov Exponent",
+    "gallery": true
```
Comment (wodeni) [inline packages/examples/src/registry.json:536]:
```suggestion
    "gallery": true
  }  
```

Comment (FuzzyBunnys) [reply packages/examples/src/registry.json:536]: oops! Thought I'd caught all the dumb mistakes with the first fix, thanks for checking it over!

File: packages/examples/src/registry.json [hunk 2]
```diff
@@ -530,5 +530,9 @@
   "alloy-models/generic": {
     "name": "Alloy: Generic Style",
     "gallery": false
-  }
+  },
+  "Dynamics/Lyapunov": {
+    "name" : "Lyapunov Exponent",
```
Comment (wodeni) [inline packages/examples/src/registry.json:535]:
```suggestion
    "name": "Lyapunov Exponent",
```

Comment (wodeni) [reply packages/examples/src/registry.json:535]: another small change to make the auto-formatter ([prettier](https://prettier.io/)) happy. We usually run `yarn format` or rely on the IDE to standardize code style. Also mentioned in the [docs](https://github.com/penrose/penrose/blob/main/CONTRIBUTING.md#opening-a-pull-request-pr)

Comment (wodeni) [reply packages/examples/src/registry.json]: totally understood! thanks for going through the troubles. excited to have your diagram in the gallery!

**Pull request merged**